memory/ndctl.py

```diff
@@ -551,6 +562,83 @@ def test_region_capabilities(self):
                     region=reg_name, mode='devdax', align=size)
                 self.plib.destroy_namespace(region=reg_name, force=True)
 
+    @avocado.fail_on(pmem.PMemException)
+    def write_infoblock(self, ns_name, align):
+        """
+        Write_infoblock on given namespace
+        """
+        if not self.plib.check_ndctl_subcmd("write-infoblock"):
+            self.cancel("Binary does not support write-infoblock")
+        write_cmd = "%s write-infoblock --align %s -m devdax "\
+                    "%s" % (self.ndctl, align, ns_name)
+        if process.system(write_cmd, ignore_status=True):
+            self.fail("write-infoblock command failed")
+        read_cmd = "%s read-infoblock -j %s" % (self.ndctl, ns_name)
+        read_out = process.system_output(read_cmd, ignore_status=True).decode()
+        read_out = json.loads(read_out)
+        if align != int(self.plib.run_ndctl_list_val(read_out[0], 'align')):
+            self.fail("Alignment has not changed")
+
+    @avocado.fail_on(pmem.PMemException)
+    def test_write_infoblock(self):
+        """
+        Test write_infoblock with align size
+        """
+        region = self.get_default_region()
+        self.plib.disable_namespace(region=region)
+        self.plib.destroy_namespace(region=region)
+        self.plib.create_namespace(region=region, mode='devdax')
+        ns_name = self.plib.run_ndctl_list_val(
+            self.plib.run_ndctl_list("-N -r %s" % region)[0], 'dev')
+        self.plib.disable_namespace(namespace=ns_name)
+        self.write_infoblock(ns_name, self.get_size_alignval())
+        failed = False
+        try:
+            self.plib.enable_namespace(namespace=ns_name)
+        except pmem.PMemException:
+            self.log.error("Enabling namespace failed")
+            failed = True
+
+        if failed:
+            self.fail("Enabling namespace failed")
```

Comment:
```
try:
    self.plib.enable_namespace(namespace=ns_name)
except pmem.PMemException:
    self.log.error("Enabling namespace failed")
    self.fail("Enabling namespace failed")
```

Reply:
Thanks, this comment actually helped me find that even try-except is not required as `avocado.fail_on(pmem.PMemException)` takes care of it. Will make this change.

Reply:
@narasimhan-v Pushed with the changes, Thanks.